### DISCUSSION TITLE
index.html not generated?

### DISCUSSION THREAD

**Discussion by lockinmarv** [frandiox/vite-ssr#38] [Q&A] [answered]
Hey! When I run `vite-ssr build` it logs the generated files to the console like `dist/client/index.html                  0.42kb` but when I look into the `dist/client` directory, the `index.html` is missing.
Is this correct and if so, how can I deploy it? Can it generate static pages?

**Comment by frandiox** [comment, marked as answer]
@MarvinRudolph Hello! The `index.html` file is removed at the end of the build so it's normal that you don't find it. It's content is inlined in the SSR build so it should work without issues. To deploy, you can see an example of a Node.js server [here](https://github.com/frandiox/vite-ssr/blob/master/examples/node-server/index.js) (so anything that works with Node.js should be fine). There's also an example deploying to Vercel [here](https://github.com/frandiox/vitesse-ssr-template/blob/master/serverless/api/index.js).

> Can it generate static pages?

There's no functionality in `vite-ssr` for static pages at the moment. You can either use:
- `vite-ssg`, which has similar API to `vite-ssr` but focuses on static pages.
- `vitedge`, which does SSR in CF workers so the result is cached at the edge (pretty much like a static file).

**Reply by lockinmarv** [reply]
Thanks for your fast answer, I will try that! Keep the superb work up. 🚀
